jiant/models.py

```diff
@@ -186,7 +187,6 @@ def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
             sep_embs_for_skip=args.sep_embs_for_skip,
             cove_layer=cove_layer,
         )
-        d_sent = 2 * args.d_hid
```

Comment:
What's this for?

Reply:
Ah this is an unnecessary change and nothing to do with this PR. I will revert.